subworkflows/local/vcf_concatenate_germline/main.nf

```diff
@@ -13,10 +14,14 @@ workflow CONCATENATE_GERMLINE_VCFS {
 
     take:
     vcfs
+    fasta
 
     main:
     versions = Channel.empty()
 
+    // Normalize vcf-files (added by jcdelmas 240415)
+    GERMLINE_VCFS_NORM(germline_vcfs_with_tbis, fasta)
```

Comment:
I think this might be causing your error. The `germline_vcfs+with_tbis` is defined further down